### DISCUSSION TITLE
After copied these language files into eclipse\plugins directory and restart BIRT, but the BIRT ui still in English:

### DISCUSSION THREAD

**Discussion by luckyju2008** [eclipse-birt/birt#1663] [Q&A] [answered]
After copied these language files into eclipse\plugins directory and restart BIRT, but the BIRT ui still in English.
Is need other settings?
![image](https://github.com/eclipse-birt/birt/assets/168630711/c8a5fc4f-4019-42b4-8d8e-be55865baca8)

**Comment by merks** [comment, marked as answer]
You should install them using Help -> Install New Software.

![image](https://github.com/eclipse-birt/birt/assets/208716/df6be647-6c42-4968-9eb7-3c6c52c5e80e)

I'm not sure that copying them into the plugins folder ever will work.  I think you could maybe put them in the dropins folder.  But installing them is the proper way. 

I think Eclipse should know your locale but there is a -nl option:

https://help.eclipse.org/latest/index.jsp?topic=%2Forg.eclipse.platform.doc.isv%2Freference%2Fmisc%2Fruntime-options.html
